packages/dai-plugin-governance/package.json

```diff
@@ -1,22 +1,21 @@
 {
   "name": "@makerdao/dai-plugin-governance",
   "description": "A dai.js plugin for adding MKR governance support to dapps.",
-  "version": "0.9.6-rc.2",
+  "version": "0.10.0",
   "license": "MIT",
   "repository": {
     "type": "git",
     "url": "https://github.com/makerdao/dai-plugin-governance.git"
   },
-  "dependencies": {
-    "@babel/runtime": "^7.4.3",
-    "@makerdao/currency": "^0.9.5",
-    "@makerdao/services-core": "^0.9.5",
-    "assert": "^2.0.0",
-    "ramda": "^0.25.0",
-    "web3-utils": "^1.0.0-beta.36"
-  },
+  "main": "dist/index.js",
+  "browser": "umd/index.js",
+  "unpkg": "umd/index.js",
+  "jsdelivr": "umd/index.js",
   "scripts": {
-    "build": "../../scripts/build.sh",
+    "build": "yarn clean && yarn build:cjs && yarn build:umd",
```

Comment:
split each build script into  a`umd` and `cjs` version for the different transpilation targets